addons/point_of_sale/static/src/js/Screens/ProductScreen/ControlButtons/OrderlineCustomerNoteButton.js

```diff
@@ -0,0 +1,42 @@
+odoo.define('point_of_sale.OrderlineCustomerNoteButton', function(require) {
+    'use strict';
+
+    const PosComponent = require('point_of_sale.PosComponent');
+    const ProductScreen = require('point_of_sale.ProductScreen');
+    const { useListener } = require('web.custom_hooks');
+    const Registries = require('point_of_sale.Registries');
+
+    class OrderlineCustomerNoteButton extends PosComponent {
+        constructor() {
+            super(...arguments);
+            useListener('click', this.onClick);
+        }
+        get selectedOrderline() {
```

Comment:
no need of getter here as not used in xml.

In the function onClick(), add 'const selectedOrderline = this.env.pos.get_order().get_selected_orderline();

Reply:
True, done